src/App.tsx

```diff
@@ -11,14 +11,17 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
   };
 
   add100 = () => {
-    // write code here
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
```

Comment:
Notes: Just for your information
Also, you can make a destructing
```suggestion
    this.setState(({ count })=> ({
      count: count + 100,
    }));
```